iym/data_package/data_package.json

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
let's rename this so it's a bit clearer that it's a template

Reply:
done